File: 백준/Bronze/1934. 최소공배수/최소공배수.py
T = int(input())

def gcd(a,b):
    while b>0:
        a, b = b, a % b
    return int(a)

def lcm(a,b):
    return int(a * b / gcd(a, b))

for i in range(T):
    A,B = map(int, input().split())
    print(lcm(A,B))

File: 백준/Bronze/1934. 최소공배수/최소공배수.cc
#include <bits/stdc++.h>
using namespace std;

int gcd(int a, int b) {
	int c = a % b;
	while (c != 0) {
		a = b;
		b = c;
		c = a % b;
	}
	return b;
}

int lcm(int a, int b) {
	return (a * b) / gcd(a, b);
}

int main(){
    ios_base::sync_with_stdio(false);
    cin.tie(0);

    int T, A, B;
    cin >> T;
    for (int i=0; i<T; i++){
        cin >> A >> B;
        cout << lcm(A, B) << '\n';
    }
    
    return 0;
}